extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -74,6 +75,11 @@ export class AmpAd extends AMP.BaseElement {
       const slotId = window.ampAdSlotIdCounter++;
       this.element.setAttribute('data-amp-slot-index', slotId);
 
+      if (this.element.getAttribute('data-multi-size')
+          && Object.keys(getAllParentAdContainers(this.element)).length != 0) {
```

Comment:
Move && to previous line.  How do we handle cases where multi-size is set to the current slot size?  What if the attribute value is an invalid format (e.g. fooxbar)?

Not sure I'm a fan of unsetting the attribute like this.  Instead, can we see how often currently publishers are using multi-size within a container?  Hopefully its very rare in which case we can just indicate its not supported and collapse the slot or better yet enforce as part of validation.